### ISSUE TITLE
move 1.4 board wifi connector location so wifi covers board area

### ISSUE THREAD

**Issue by RevoluPowered** [bigtreetech/BIGTREETECH-SKR-V1.3#280]
I'm using the SKR 1.4 Turbo I would like if the WiFi connection on the next revision of these boards had the WiFi ESP-01s connector in another place.

The reason for this would be so that you can use an LCD and the WiFi without making up a IDC cable to handle the ESP01S.

Right now the orientation of the ESP01S is out of the board, which then means it hits the case in most set ups, as the ESP01S has to be facing away from the board and the SD Card slot is on this side meaning it hits the case if you make a mount for it.

As the other IO is on that side too this means a case is hard to fit for this, without making an IDC cable up for the ESP01S.

Would also be nice if the board could have a higher current mosfet, and perhaps an optional heatsink mounting screws for the board so we can cool the drivers and the board.

**Comment by andrewfrewer** [comment]
I agree. It would be best to not have the wifi module sticking off the board.
